### DISCUSSION TITLE
Unable to upload or change template images.

### DISCUSSION THREAD

**Discussion by wade-lich** [opendcim/openDCIM#1379] [Q&A] [answered]
problem exists on the demo site.

after choosing a file to upload, the file does not transfer or render in the uploader.

existing template images cannot be changed by clicking on the image.  The list presents but will not allow a selection to be made.  

I have rebuilt the cache server times.

**Comment by wilpig** [comment, marked as answer]
https://github.com/opendcim/openDCIM/issues/1373
https://github.com/opendcim/openDCIM/issues/1374
https://github.com/opendcim/openDCIM/commit/a65a3d1de007eb103cca0351d888bee7d85a6d53

Problem has been fixed

**Comment by wilpig** [comment]
You can see the fix in place at http://dev.opendcim.org/device_templates.php .  See the difference between the two sites on the page loading.

<img width="588" alt="Screenshot 2023-06-14 at 6 58 53 PM" src="https://github.com/opendcim/openDCIM/assets/1653882/58eea08a-2648-4ff2-bea0-25e37f12a9ff">

^-- see how not all of the fields loaded compared to the dev site below
<img width="847" alt="Screenshot 2023-06-14 at 6 59 00 PM" src="https://github.com/opendcim/openDCIM/assets/1653882/b3299ee2-c677-42fd-8c03-cd3f56f5478e">
